pkgs/applications/misc/tdrop/default.nix

```diff
@@ -1,24 +1,34 @@
 { stdenv, lib, fetchFromGitHub, makeWrapper
-, xwininfo, xdotool, xprop }:
+, xwininfo, xdotool, xprop, gawk, coreutils
+, gnugrep, procps-ng }:
 
-stdenv.mkDerivation {
+stdenv.mkDerivation rec {
   pname = "tdrop";
-  version = "unstable-2018-11-13";
+  version = "unstable-2019-10-04";
 
   src = fetchFromGitHub {
     owner = "noctuid";
     repo = "tdrop";
-    rev = "198795c0d2573a31979330d6a2ae946eb81deebf";
-    sha256 = "1fhibqgmls64mylcb6q46ipmg1q6pvaqm26vz933gqav6cqsbdzs";
+    rev = "60435d240f47719a69ab1f7b2be68ca3ef1df1df";
+    sha256 = "1krry5p17hzp0lpilylch8v1f485kgxkx4lmwpghzj4vpgvwk8k4";
   };
 
   dontBuild = true;
 
   installFlags = [ "PREFIX=$(out)" ];
 
-  postInstall = ''
-    wrapProgram $out/bin/tdrop \
-      --prefix PATH : ${lib.makeBinPath [ xwininfo xdotool xprop ]}
+  postInstall = let
+    binPath = lib.makeBinPath [
+      xwininfo
+      xdotool
+      xprop
+      gawk
+      coreutils
+      gnugrep
```

Comment:
Do we definitely need these very standard tools in the `$PATH` as well? It impacts a little bit on the closure size

Reply:
They're used in the script. Sorry, I don't really understand what you suggest.

Reply:
I mean that these programs are in the `stdenv`'s `$PATH` and in (probably) every user's `$PATH` - hence it shouldn't be necessary to pin them in your derivation just for the sake of making sure such an obvious dependency exists on the system. I suggest removing these 3 lines and hoping that within the lines of regular day-to-day usage, users won't complain.